BUILDME.md

```diff
@@ -0,0 +1,218 @@
+# Branch Metrics C++ SDK
+
+[![License](https://img.shields.io/badge/license-MIT-green.svg?style=flat)](https://github.com/BranchMetrics/cpp-branch-deep-linking-attribution/blob/master/LICENSE)
+[![CircleCI](https://img.shields.io/circleci/project/github/BranchMetrics/cpp-branch-deep-linking-attribution.svg)](https://circleci.com/gh/BranchMetrics/cpp-branch-deep-linking-attribution)
+
+[Platform prerequisites]: #platform-prerequisites
+[Install CMake and Conan]: #install-cmake-and-conan
+[Set up Branch remote]: #set-up-branch-remote
+[Set up a default conan profile]: #set-up-a-default-conan-profile
+[Integration]: #integration
+[Sample apps]: #sample-apps
+
+## Contents
+- [Platform prerequisites]
+- [Install CMake and Conan]
+- [Set up a default Conan profile]
+- [Set up Branch remote]
+- [Integration]
+- [Sample apps]
+
+## Platform prerequisites
+
+### Windows
+
+- Visual Studio 2019
+- Python/Pip (can be installed through Visual Studio or otherwise)
+- (Optional): Conan Visual Studio Extension
+- Ensure that python, pip and the pip installation bin folder are in your PATH.
+
+### macOS
```

Comment:
@apeterson-branch @csalmi-branch I don't think we should be adding anything related to Mac OS or any other OS we haven't explicitly tested the C++ SDK on (saw Linux in these docs, too) to these docs. 

I know that this is theoretically a cross-platform SDK, but until we verify that it works the way it's supposed to on other OSes (which we have not), we shouldn't be billing it as such.